pandas/io/tests/test_sql.py

```diff
@@ -236,7 +236,7 @@ def _close_conn(self):
         pass
 
 
-class PandasSQLTest(unittest.TestCase):
+class PandasSQLTest(object):
```

Comment:
I see! Sorry. 
FWIW I've generally called those `...Base` rather than `...Test`, to make that explicit. But only a personal preference.